Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -151,7 +137,18 @@ void WebXROpaqueFramebuffer::startFrame(const PlatformXR::Device::FrameData::Lay
     // Tell the GraphicsContextGL to use the IOSurface as the backing store for m_opaqueTexture.
     if (data.isShared) {
 #if !PLATFORM(IOS_FAMILY_SIMULATOR)
-        m_ioSurfaceTextureHandle = gCGL->attachIOSurfaceToSharedTexture(textureTarget, data.surface.get());
+        auto surfaceTextureAttachment = gCGL->attachIOSurfaceToSharedTexture(textureTarget, data.surface.get());
+        if (!surfaceTextureAttachment) {
+            gCGL->deleteTexture(m_opaqueTexture);
+            m_opaqueTexture = 0;
+            return;
+        }
+
+        auto [textureHandle, textureWidth, textureHeight] = surfaceTextureAttachment.value();
```

Comment:
Maybe this should be `*surfaceTextureAttachment`. I like `.value()` more though.

Reply:
I prefer to use `value()` on `std::optional()` over `operator*()`

Reply:
Disagree, but it’s definitely a taste thing.